OrbitCore/Path.cpp

```diff
@@ -56,15 +56,15 @@ std::string Path::GetExecutablePath() {
 }
 
 bool Path::FileExists(const std::string& file) {
```

Comment:
can you add a test for this?

Reply:
Added very basic tests.

Reply:
Ok, so what qualifies as a file? Is a character device a file? Is a directory a file?